src/itemData/index.tsx

```diff
@@ -9,7 +9,7 @@ export interface ItemDataProps extends A11yProps {
   readonly data: string | JSX.Element
   readonly dataStrikeThrough?: boolean
   readonly dataAriaLabel?: string
-  readonly mainInfo: string
+  readonly mainInfo: React.ReactNode
   readonly className?: string
   readonly mainTitle?: string
```

Comment:
Should we do the same for `mainTitle`?

Reply:
If we do so we should update the base item also `readonly leftTitle?: string`